xcode/Subconscious/Subconscious.xcodeproj/project.pbxproj

```diff
@@ -2327,8 +2335,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/gordonbrander/ObservableStore";
 			requirement = {
-				kind = upToNextMajorVersion;
-				minimumVersion = 0.3.0;
+				kind = exactVersion;
```

Comment:
Locking in this dependency too, while I'm at it.

ObservableStore has some breaking changes in the upcoming 0.4.0. Better to upgrade when we choose to.